builder/bin/image_gen.rs

```diff
@@ -19,6 +19,14 @@ fn main() {
                 .value_parser(value_parser!(PathBuf)),
         )
         .arg(arg!(--"fw" [FILE] "FW bundle image").value_parser(value_parser!(PathBuf)))
+        .arg(
+            arg!(--"fmc-svn" [FILE] "Security Version Number of FMC firmware image")
+                .value_parser(value_parser!(u32)),
+        )
+        .arg(
+            arg!(--"rt-svn" [FILE] "Security Version Number of RT firmware image")
```

Comment:
```suggestion
            arg!(--"rt-svn" [VALUE] "Security Version Number of RT firmware image")
```

nit: Similar to the above comment

Reply:
Done